src/test/java/uk/gov/hmcts/ccd/data/caseaccess/AMCaseUserRepositoryComponentTest.java

```diff
@@ -0,0 +1,121 @@
+package uk.gov.hmcts.ccd.data.caseaccess;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.ccd.BaseTest;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.List;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.collection.IsIterableContainingInAnyOrder.containsInAnyOrder;
+
+@Transactional
+@Ignore("This is pending AM library implementation")
```

Comment:
This is to be implemented by subsequent story that implements the adapter/mapper from/to AM DTO.